src/main/java/uk/ac/ic/wlgitbridge/server/Oauth2Filter.java

```diff
@@ -135,15 +139,15 @@ private void getAndInjectCredentials(
                                     servletResponse
                             );
                         } else {
-                            unauthorized(projectName, response);
+                            unauthorized(projectName, capturedUsername, 0, request, response);
```

Comment:
Perhaps we can use the 401 from the response we give in `unauthorized()`?

Reply:
@winstonli My intention here was to return the status code of the OAuth request to Overleaf, which I think I'm doing on line 127.  If I'm reading the code right, if we reach the `unauthorized` on line 140 then the OAuth request hasn't been made, so I'm arbitrarily showing `0` just to reflect that this is a different sort of `unauthorized`.

Reply:
Ok, that makes perfect sense.

Reply:
OK, thanks 👍